Master-OB-OpenAPI.json

```
@@ -7390,15 +7390,15 @@
           }
         ]
       },
-      "SolarCellQuantity": {
+      "CellCount": {
         "allOf": [
           {
             "$ref": "#/components/schemas/TaxonomyElementNumber"
           },
           {
             "type": "object",
-            "description": "The number of ProdSolarCell in a module",
-            "x-ob-item-type": "IntegerItemType",
+            "description": "The number of ProdCell in a module",
+            "x-ob-item-type": "DecimalItemType",
```

```suggestion
            "description": "The number of cells in a module",
            "x-ob-item-type": "IntegerItemType",
```

This should stay as an integer. For a module with half-cut cells, use CellCutType with a value in CellCutTypeItemType.